app/migrations/Version20180921144421.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Migrations\SkipMigrationException;
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180921144421 extends AbstractMauticMigration
+{
+    /**
+     * @param Schema $schema
+     *
+     * @throws SkipMigrationException
+     * @throws \Doctrine\DBAL\Schema\SchemaException
+     */
+    public function preUp(Schema $schema)
+    {
+        if ($schema->getTable($this->prefix.'lead_fields')->hasColumn('column_is_not_created')) {
```

Comment:
Should be hasTable

Failed during migrations

`Migration 20170516000000 failed during Pre-Checks. Error An exception occurred while executing 'SELECT t0.is_published AS is_published_1, t0.date_added AS date_added_2, t0.created_by AS created_by_3, t0.created_by_user AS created_by_user_4, t0.date_modified AS date_modified_5, t0.modified_by AS modified_by_6, t0.modified_by_user AS modified_by_user_7, t0.checked_out AS checked_out_8, t0.checked_out_by AS checked_out_by_9, t0.checked_out_by_user AS checked_out_by_user_10, t0.id AS id_11, t0.label AS label_12, t0.alias AS alias_13, t0.type AS type_14, t0.field_group AS field_group_15, t0.default_value AS default_value_16, t0.is_required AS is_required_17, t0.is_fixed AS is_fixed_18, t0.is_visible AS is_visible_19, t0.is_short_visible AS is_short_visible_20, t0.is_listable AS is_listable_21, t0.is_publicly_updatable AS is_publicly_updatable_22, t0.is_unique_identifer AS is_unique_identifer_23, t0.field_order AS field_order_24, t0.object AS object_25, t0.properties AS properties_26, t0.column_is_not_created AS column_is_not_created_27, t0.original_is_published_value AS original_is_published_value_28 FROM lead_fields t0 WHERE t0.alias = ? LIMIT 1' with params ["points"]:`